apps/web/app/WithEmbedSSR.tsx

```diff
@@ -14,7 +14,10 @@ export default function withEmbedSsrAppDir<T extends Record<string, any>>(
 ) {
   return async (context: GetServerSidePropsContext): Promise<T> => {
     const { embed, layout } = context.query;
-
+    const isCOEPEnabled = context.query["flag.coep"] === "true";
+    if (isCOEPEnabled) {
+      context.res.setHeader("Cross-Origin-Embedder-Policy", "require-corp");
```

Comment:
Setting it unconditionally will break "pushing events to certain third party analytics" apps. So, we let the embed user opt-in by flag.coep=true